Backend/app/provider-platform/dynamic-offer-driver-app/Main/src/Domain/Action/UI/Ride/CancelRide/Internal.hs

```diff
@@ -73,7 +76,11 @@ cancelRideImpl ::
     HasShortDurationRetryCfg r c,
     HasField "maxShards" r Int,
     HasCacheConfig r,
-    HasFlowEnv m r '["nwAddress" ::: BaseUrl]
+    HasFlowEnv m r '["nwAddress" ::: BaseUrl],
+    HasFlowEnv m r '["kafkaProducerTools" ::: KafkaProducerTools],
+    HasField "version" r r0,
+    HasField "getDeploymentVersion" r0 Text,
+    HasField "eventStreamMap" r [EventStreamMap]
```

Comment:
this can be simplified with some synonym:
`type EventStreamFlow = HasFlowEnv m r ["kafkaProducerTools" ::: KafkaProducerTools, "getDeploymentVersion"::: DeploymentVersion, "eventStreamMap" ::: [EventStreamMap]],`

Reply:
Done